### PR TITLE
Feature/account subscriptions

### PR DESCRIPTION
**Pull request by matjazzorko**

### CODE REVIEW

File: src/sections/customer-account-header.liquid [hunk 1]
```diff
@@ -73,7 +73,7 @@
                     <span class="account-header__status-icon" aria-ignore="true">
                       <svg width="12" height="15" viewBox="0 0 12 15" xmlns="http://www.w3.org/2000/svg"><g fill="none" fill-rule="evenodd"><path d="M6 1.04C2.574 4.366.75 7.003.75 9c0 1.45.588 2.762 1.538 3.712A5.234 5.234 0 006 14.25c1.45 0 2.762-.588 3.712-1.538A5.234 5.234 0 0011.25 9c0-1.997-1.824-4.634-5.25-7.96z" stroke="#FFF" stroke-width="1.5"/><path fill="#FFF" d="M6 10.5l-1.763.927.336-1.963-1.426-1.391 1.971-.287L6 6l.882 1.786 1.971.287-1.426 1.391.336 1.963z"/></g></svg>
                     </span>
-                    <span class="account-header__status-number">300</span>
+                    <span class="account-header__status-number" id="stamped-rewards-points-placeholder">300</span>
```
Comment (jernejdroljc) [inline src/sections/customer-account-header.liquid:76]: This is hardcoded value, is this ok?

File: src/snippets/mydyln-script.liquid [hunk 2]
```diff
@@ -57,17 +57,17 @@ $(function(){
...
-//  console.log('Button Text:'+ $('#subscription-wrap .custom-select-styled').text());  
-//$('#subscription-wrap .custom-select-styled').click();  
-//$('#subscription-wrap .custom-select-option').find('button[rel="3"]').click();
-//console.log('selected '+$('#subscription-wrap .custom-select-option').find('button[rel="3"]').attr('rel'));
-  
+
+  // change to subscribe option
+  $('#subscription-wrap .diffusers__sub-btn').click();
+  $('#subscription-wrap .diffusers__sub-back').html('<strong>Not ready for a Subscription?</strong><br><span>Shop one time purchase:</span>');
+  //  console.log('Button Text:'+ $('#subscription-wrap .custom-select-styled').text());
```
Comment (jernejdroljc) [inline src/snippets/mydyln-script.liquid:64]: Remove comments if they are not necessary.

File: src/styles/6.components/global/global.scss [hunk 3]
```diff
@@ -14,3 +14,448 @@
 @import "./diffusers";
 @import "./pagination";
 @import "./challenge";
+@import "./contact-sidebar";
+
+/*
```
Comment (jernejdroljc) [inline src/styles/6.components/global/global.scss:19]: Remove comments if not necessary.

**Comment by dejanstrancar** [comment]
There is bounch of console logs that needs to be removed before merging.

**Comment by dejanstrancar** [comment]
@mazdihasan read my comment above with console.logs also its not a good practice to put styles and script directly inside liquid files.

**Pull request merged**